src/components/course/course-header/tests/CourseRunCard.test.jsx

```diff
@@ -52,19 +93,67 @@ const CourseRunCardWrapper = (props) => {
     userCanRequestSubsidyForCourse: mockUserCanRequestSubsidy,
   };
   return (
-    <CourseContext.Provider value={courseContextValue}>
-      <CourseRunCard
-        courseRun={mockCourseRun}
-        {...props}
-      />
-    </CourseContext.Provider>
+    <QueryClientProvider client={queryClient()}>
+      <CourseContext.Provider value={courseContextValue}>
```

Comment:
Is this `CourseContext.Provider` necessary? Looks like its only using API data, not the `algoliaSearchParams`; likely could be removed.